tests/getServerSideProps.test.ts

```diff
@@ -0,0 +1,97 @@
+import { seal, defaults } from "@hapi/iron";
+import { IncomingMessage, ServerResponse } from "http";
+import fetch from "jest-fetch-mock";
+import { enableFetchMocks } from "jest-fetch-mock";
+import jwt from "jsonwebtoken";
+import { Socket } from "net";
+
+import { getServerSideProps } from "../src/pages/account";
+import {
+  mockedSortedResponse,
+  mockedResponse,
+} from "./__mocks__/managementResponse";
+
+enableFetchMocks();
+
+describe("getServerSideProps", () => {
+  beforeEach(() => {
+    fetch.resetMocks();
+  });
+
+  afterEach(() => {
+    mockedContext.req.headers = {};
+
+    mockedContext.req.rawHeaders = [];
+  });
+
+  const mockedRequest = new IncomingMessage(new Socket());
+
+  const mockedContext = {
```

Comment:
I would prefer to have the constants defined before the `beforeEach` that uses them